.gitattributes

```diff
@@ -0,0 +1 @@
+node_modules/* text eol=lf
```

Comment:
please provide dockerFile for mongoDb or host it on for example AWS and create for everyone each collection..


Also prvodie some migration code for DB models you added